### ISSUE TITLE
Regression after 1.58, docker-compose.override.yml not supported anymore

### ISSUE THREAD

**Issue by Yehonal** [microsoft/vscode-remote-release#5355]
This issue regarding the support of docker-compose.override.yml has been closed after vscode 1.57 https://github.com/microsoft/vscode-remote-release/issues/1080 by using an empty array as value of `dockerComposeFile`. However, after latest 1.58 version, that property cannot be an empty array anymore otherwise this error will be thrown:

```
[4254 ms] Error: Could not read file: undefined
        at Object.readFile (c:\Users\roy_w\.vscode\extensions\ms-vscode-remote.remote-containers-0.187.0\dist\extension\extension.js:115:5818)
        at async Hte (c:\Users\roy_w\.vscode\extensions\ms-vscode-remote.remote-containers-0.187.0\dist\extension\extension.js:250:6)
        at async YF (c:\Users\roy_w\.vscode\extensions\ms-vscode-remote.remote-containers-0.187.0\dist\extension\extension.js:240:449)
        at async CZ (c:\Users\roy_w\.vscode\extensions\ms-vscode-remote.remote-containers-0.187.0\dist\extension\extension.js:220:10180)
```

- VSCode Version: 1.58.2
- Local OS Version: Windows with WSL
- Remote OS Version: Ubuntu
- Remote Extension/Connection Type: Docker & WSL
- Logs:

Steps to Reproduce:

1. set `"dockerComposeFile": [],` in devcontainer.json
2. rebuild

**Comment by Yehonal** [comment]
Related? https://github.com/microsoft/vscode-remote-release/issues/5310

**Comment by Yehonal** [comment]
closed in 1.187.1
